### DISCUSSION TITLE
Settings Button in the Header

### DISCUSSION THREAD

**Discussion by faizanraso** [supabase/supabase#17819] [Feature Requests]
### Resolved: [easily accessible on the left hand side under Account > Preferences](https://github.com/orgs/supabase/discussions/17819#discussioncomment-7169575)

---

I really want a way to access account settings easily. I'm honestly not even sure how to do it right now, but a way to change settings like email, user name, theme preference etc. by having some button in the header would be super useful.

     

<img width="1512" alt="Screenshot 2023-09-29 at 7 57 04 PM" src="https://github.com/supabase/supabase/assets/59986120/4f7552ec-5d3c-4a11-9b86-d1fb23aeed50">

**Comment by w3b6x9** [comment]
It's on the left hand side: `Account > Preferences`

<img width="1045" alt="Screenshot 2023-10-02 at 17 40 27" src="https://github.com/supabase/supabase/assets/5532241/c19ebf1d-fbb0-498c-bd02-8fd16ea09ac0">

**Reply by faizanraso** [reply]
Oh I see. Guess I'm just used to it being on the header on other apps that I didn't pay attention to the sidebar. Thanks @w3b6x9 !
